src/handlers/store-standardised-invoices/get-standardised-invoice/get-standardised-invoice-default.ts

```diff
@@ -37,17 +42,35 @@ export const getStandardisedInvoiceDefault: StandardisationModule = (
     tax_payer_id: getTaxPayerId(summaryFields),
   };
 
-  const standardisedLineItems = lineItems.map((item) => {
-    const itemFields = item.LineItemExpenseFields ?? [];
+  const serviceRegex = new RegExp(serviceRegexPattern, "i");
+
+  const standardisedLineItems = lineItems.reduce<StandardisedLineItem[]>(
+    (acc, item) => {
+      const itemFields = item.LineItemExpenseFields ?? [];
+
+      const itemDescription = getItemDescription(itemFields);
+      console.log(
+        "🚀 ~ file: get-standardised-invoice-default.ts:52 ~ itemDescription",
+        itemDescription
+      );
```

Comment:
Maybe this console.log was left in unintentionally?

Reply:
oops yep it surely was